### ISSUE TITLE
Cannot convert a Symbol value to a string when compile with NODE_ENV=production

### ISSUE THREAD

**Issue by ta369e** [inertiajs/inertia#731]
### Versions:

- `@inertiajs/inertia` version:^0.9.2
- `@inertiajs/inertia-vue3` version:^0.4.4

### Describe the problem:

after upgrading to the latest version to use inertia-head, and when I ONLY compile npm with  NODE_ENV=production
title meta not showing up, and the following console error appears:
```
app.js?id=7136e0f4ca739d2a8440:2 TypeError: Cannot convert a Symbol value to a string
    at Proxy.renderTagStart (...)
    at Proxy.renderTag (...)
    at ...
    at Array.map (<anonymous>)
    at Proxy.renderNodes (...)
    at Proxy.render (...)
    at Cr (...)
    at (...)
    at r (...)
    at le (...)
```
### Steps to reproduce:

Layout:
```
<template>
    <div>
        <j-meta />
        <navbar />
        <div class="appBody">
            <sidbar :current="route().current()"/>
            <div class="content inner-content">
                <session />
                <div>
                    <slot />
                </div>
            </div>
        </div>
        <j-footer />
    </div>
</template>

<script>
import Sidbar from '@/Components/layout/Sidebar';
import Navbar from '@/Components/layout/Navbar';
import Session from '@/Components/layout/Session';
import JFooter from '@/Components/layout/Footer';

export default {
    components: {
        Sidbar,
        Navbar,
        JFooter,
        Session
    }
};
</script>
```

Page: 
```
<template>
    <div class="container-fluid">
        <j-meta title="dashboard" />
    </div>
</template>

<script>
import Authenticated from '@/Layouts/Authenticated';

export default {
    layout: Authenticated,
};
</script>
```

J-META:
```
<template>
    <inertia-head :title="title ? `xx | ${title}` : 'xx'">
        <slot/>
    </inertia-head>
</template>

<script>
export default {
    props: {
        title: String,
    },
};
</script>
```

COMMAND:
```
npm run production
```

**Comment by reinink** [comment]
Fixed (#731) and [released](https://inertiajs.com/releases/inertia-vue3-0.4.5-2021-06-08).

Thanks for reporting. 👍

**Comment by valh1996** [comment]
@reinink Hi, the error seems to be back in version >= 1.0.0.

Indeed, it looks like having a <slot /> triggers the error.
Any workaround please?
